packages/cli-test/src/cli/commands/platform.spec.ts

```diff
@@ -86,6 +87,12 @@ describe('platform commands', () => {
         sinon.match.string,
         sinon.match.array.contains(['run', '--cleanup', '--hide-triggers']),
       );
+      sandbox.assert.calledWith(
+        waitForOutputSpy,
+        SlackTracerId.SLACK_TRACE_PLATFORM_RUN_START,
```

Comment:
🗣️ note: This is preferred over `SLACK_TRACE_PLATFORM_RUN_READY` since the "ready" variable appears once activity logs stream for CLI managed connections. It does not appear for `bolt` apps at this time.

👁️‍🗨️ thought: I'm unsure if we should add that trace to the CLI outputs in a follow up PR? For now I think using **a** trace is still an improvement but I'm somewhat worried about tests skipping to the next step before `run` is settled for some apps... 🦕

Reply:
> I'm unsure if we should add that trace to the CLI outputs in a follow up PR?

Just checking if I understand correctly - are you thinking that the CLI would print the `SLACK_TRACE_PLATFORM_RUN_READY` trace for sdk-managed socket connections?